lib/miio/Dummycloud.js

```diff
@@ -97,7 +97,7 @@ const Dummycloud = function (options) {
                         break;
                     case "props":
                         Object.assign(this.connectedRobot.status, decodedResponse.msg.params);
-                        this.events.emit("miio.status", this.connectedRobot.status);
+                        //this.events.emit("miio.status", this.connectedRobot.status);
```

Comment:
So, just comment out and that's it?

Reply:
Props send you every 30 minutes the status of the robot, i can't see any reason why i need this. There is no change in this status messages. When the robot start the cleaning, error, etc. it sends `miio.status`